pkg/component/etcd/statefulset/statefulset.go

```diff
@@ -157,7 +157,7 @@ func (c *component) createDeployFlow(ctx context.Context) (*flow.Flow, error) {
 		return nil, err
 	}
 
-	flowName := fmt.Sprintf("(etcd: %s) Deploy Flow for StatefulSet %s for Namespace: %s", c.values.EtcdUID, c.values.Name, c.values.Namespace)
+	flowName := fmt.Sprintf("(etcd: %s) Deploy Flow for StatefulSet %s for Namespace: %s", c.values.OwnerReference.UID, c.values.Name, c.values.Namespace)
```

Comment:
instead of using OwnerReference.UID should you not use OwnerReference.Name instead? A UID will have a value like `058ba68b-5758-4a1a-9e88-d56b463838ad` and is hard to use to debug. If you wish to have UID then have both` <name>:<uid>`

Reply:
There are other places where you have only chosen to log OwnerReference.UID. Please check all such places.

Reply:
Fixed - https://github.com/gardener/etcd-druid/pull/559/commits/881f21af25c4a8b466c2c8f76bc51bc8b1492601